service-tests/npm.js

```diff
@@ -3,9 +3,11 @@
 const Joi = require('joi');
 const ServiceTester = require('./runner/service-tester');
 const { isSemver } = require('./helpers/validators');
+const colorscheme = require('../lib/colorscheme.json');
 
 const t = new ServiceTester({ id: 'npm', title: 'NPM' });
 module.exports = t;
+const colorsB = Object.assign({}, ...Object.keys(colorscheme).map(color => ({ [color]: colorscheme[color].colorB })));
```

Comment:
This is another good place to use `mapValues`.

Reply:
Done just before you wrote it. Thanks!